android/filament-android/src/main/java/com/google/android/filament/Stream.java

```diff
@@ -33,13 +33,32 @@
     private long mNativeObject;
     private long mNativeEngine;
 
+    /**
+     * Represents the immutable stream type.
+     *
+     * - NATIVE streams are not synchronized but are copy-free.
```

Comment:
Put the individual explanations on the enum values (each enum value can have its own javadoc body).